monarchmoney/monarchmoney.py

```diff
@@ -2481,21 +2481,27 @@ async def gql_call(
             document=graphql_query, operation_name=operation, variable_values=variables
         )
 
-    def save_session(self, filename: str) -> None:
+    def save_session(self, filename: str = None) -> None:
         """
         Saves the auth token needed to access a Monarch Money account.
         """
+        if filename is None:
+            filename = self._session_file
+        filename = os.path.abspath(filename)
+
         session_data = {"token": self._token}
-        if not os.path.exists(SESSION_DIR):
-            os.makedirs(SESSION_DIR)
 
+        os.makedirs(os.path.dirname(filename), exist_ok=True)
         with open(filename, "wb") as fh:
             pickle.dump(session_data, fh)
 
-    def load_session(self, filename: str = SESSION_FILE) -> None:
+    def load_session(self, filename: str = None) -> None:
```

Comment:
Change the type to 'Optional[str]', but keep the default as is, since it will break it for existing users. 

(There are about 700 downloads/wk of this library from PyPI!)

Reply:
Are you sure a default of `None` will break for existing users?

In the next couple lines, this bit:

```python
if filename is None:
    filename = self._session_file
```

Should handle setting the filename correctly for existing users who don't provide an explicit filename, keeping the previous behavior.

Reply:
You're right, @keegandahm , it still works!  Please make sure to rebase & merge onto origin/main to avoid any merge conflicts for the PR.

Once that's done, I'll merge it in!

Reply:
I rebased my branch on master, it should merge cleanly. Thank you for approving!